content/docs/run-api-server/ingestion.mdx

```diff
@@ -162,15 +162,15 @@ export RETRY_BACKOFF_SECONDS=20
 If Horizon was previously running, first ensure it is stopped. Then, run the following commands in parallel:
 
 1. `stellar-horizon db reingest range --parallel-workers=64 1 16999999`
-2. `stellar-horizon db reingest range --parallel-workers=24 17000000 <latest_ledger>`
+2. `stellar-horizon db reingest range --parallel-workers=20 17000000 <latest_ledger>`
 
 (Where you can find `<latest_ledger>` under [SDF Horizon's](https://horizon.stellar.org/) `core_latest_ledger` field.)
 
-When saturating an RDS instance with 15K IOPS capacity:
+When saturating a database instance with 15K IOPS capacity:
 
 (1) should take a few hours to complete.
 
-(2) should take about 1.5 days to complete.
+(2) should take about 3 days to complete.
```

Comment:
This is an unfortunate change :disappointed:

Reply:
Yes definitely! We have significantly more ledgers now and the network had been very busy in recent months which I'm sure contributed to this as well.